notes/lecture7.tex

```diff
@@ -0,0 +1,137 @@
+\section{Lecture 7: Nesterov’s accelerated gradient descent}
+
+In this lecture, we present Nesterov's accelerated gradient descent for smooth convex functions. We will derive an algorithm that obtains a runtime of $\cO \left(\frac{\beta}{t^2}\right)$ for $\beta$-smooth functions. For functions which are $\alpha$-strongly convex, we will achieve a rate of $\cO \left(\exp\left( -\sqrt{\frac{\beta}{\alpha}} t \right) \right)$. The algorithm proceeds iteratively as follows. 
+\begin{align*}
+x_0 &= y_0 = z_0, \\
+y_k &= x_k - \frac{1}{\beta} \nabla f(x_k), \\
+z_k &= z_{k -1} - \eta \|\nabla f(x_k)\|, \\
+x_{k+1} &= \tau z_k + (1 - \tau) y_k.
```

Comment:
It might be helpful to pu the `x_{k+1}` definition in the first line, since this confused people.